filesystem/sbin/start_runit

```diff
@@ -13,4 +13,4 @@ fi
 # Export the nodename set by the startup procedure. 
 export NODENAME=$(cat /var/lib/calico/nodename)
 
-exec /sbin/runsvdir -P /etc/service/enabled
+exec /usr/bin/runsvdir -P /etc/service/enabled
```

Comment:
This changed due to switching to Debian, the default path for `runsvdir` is different on a Debian install.